components/organisms/notification/NotificationItem.tsx

```diff
@@ -0,0 +1,72 @@
+import { Span } from '@/components/atoms';
+import { useCalcRegisterDate } from '@/hooks/index';
+import { NOTIFICATION_LIST } from '@/constants/index';
+import styled from 'styled-components';
+import { Layout } from '@/styles/theme';
+import { useRouter } from 'next/router';
```

Comment:
클린코드 원칙 중 하나인데 외부 라이브러리는 위로, 내부 라이브러리는 밑으로 구분해주시면 가독성에 좋아요
```suggestion
import { useRouter } from 'next/router';
import styled from 'styled-components';

import { Span } from '@/components/atoms';
import { useCalcRegisterDate } from '@/hooks/index';
import { NOTIFICATION_LIST } from '@/constants/index';
import { Layout } from '@/styles/theme';
```

Reply:
오 이런것은 몰랐네요! 👍 앞으로는 임포팅에도 신경써서 코드작성해야겠네용 🤓